client/src/components/CategoryShowPage.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from "react"
+import ProductTile from "./ProductTile.js"
+
+const CategoryShowPage = props => {
+    const [category, setCategory] = useState({
+        id: "",
+        name: "",
+        products: []
+    })
+    const categoryId = props.match.params.categoryId
```

Comment:
The indentation varies in this file compared to others. Make sure everyone on your team has their `tab size: 2` in VSCode